Lib9c/Model/Skill/AttackSkill.cs

```diff
@@ -61,7 +61,9 @@ protected AttackSkill(SkillSheet.Row skillRow, int power, int chance) : base(ski
                             isCritical = caster.IsCritical(isNormalAttack);
                             if (isCritical)
                             {
-                                damage = (int) (damage * CharacterBase.CriticalMultiplier);
+                                var critMultiplier =
+                                    Math.Max(1m, CharacterBase.CriticalMultiplier + (caster.CDMG / 10000m));
```

Comment:
```suggestion
                                    Math.Max(1.5m, CharacterBase.CriticalMultiplier + (caster.CDMG / 10000m));
```

Reply:
This is intended because crit damage can be decreased with debuffs.